README.md

```diff
@@ -1,59 +1,57 @@
-This project was bootstrapped with
-[Create React App](https://github.com/facebook/create-react-app).
+# Bench Routes Dashboard
 
-## Available Scripts
+Bench Routes Dashboard is a web UI that enables the user to access, visualize, and chart the stored data. This files explains how to setup and work with Bench Routes Dashboard.
```

Comment:
Bench routes dashboard is a user interface to access, visualize and introspect data curated by bench-routes(link-to-bench-routes).